source/tutorial/simple-component.md

```diff
@@ -202,106 +202,112 @@ Our component integration test will test two different behaviors:
-  integration: true
-});
+module('Integration | Component | rental listing', function (hooks) {
+  setupRenderingTest(hooks);
 
-test('should display rental details', function(assert) {
+  test('should display rental details', async function(assert) {
 
-});
+  });
 
-test('should toggle wide class on click', function(assert) {
+  test('should toggle wide class on click', async function(assert) {
 
-});
-test('it renders', function(assert) {
+  });
 
-  // Set any properties with this.set('myProperty', 'value');
-  // Handle any actions with this.on('myAction', function(val) { ... });
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
 
-  this.render(hbs`{{rental-listing}}`);
+    await render(hbs`{{rental-listing}}`);
 
-  assert.equal(this.$().text().trim(), '');
+    assert.equal(this.element.textContent.trim(), '');
 
-  // Template block usage:
-  this.render(hbs`
-    {{#rental-listing}}
-      template block text
-    {{/rental-listing}}
-  `);
+    // Template block usage:
+    await render(hbs`
+      {{#rental-listing}}
+        template block text
+      {{/rental-listing}}
+    `);
 
-  assert.equal(this.$().text().trim(), 'template block text');
+    assert.equal(this.element.textContent.trim(), 'template block text');
+  });
 });
 ```
 
 For the test we'll pass the component a fake object that has all the properties that our rental model has.
 We'll give the variable the name `rental`, and in each test we'll set `rental` to our local scope, represented by the `this` object.
 The render template can access values in local scope.
 
-```tests/integration/components/rental-listing-test.js{+6,+7,+8,+9,+10,+11,+12,+13,+20,+24}
-import { moduleForComponent, test } from 'ember-qunit';
+```tests/integration/components/rental-listing-test.js{+8,+9,+10,+11,+12,+13,+14,+15,+21,+25}
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
 import EmberObject from '@ember/object';
 import { run } from '@ember/runloop';
 
-let rental = EmberObject.create({
-  image: 'fake.png',
-  title: 'test-title',
-  owner: 'test-owner',
-  category: 'test-type',
-  city: 'test-city',
-  bedrooms: 3
-});
-
-moduleForComponent('rental-listing', 'Integration | Component | rental listing', {
-  integration: true
-});
-
-test('should display rental details', function(assert) {
-  this.set('rentalObj', rental);
-});
-
-test('should toggle wide class on click', function(assert) {
-  this.set('rentalObj', rental);
+module('Integration | Component | rental listing', function (hooks) {
+  setupRenderingTest(hooks);
+
+  let rental = EmberObject.create({
+    image: 'fake.png',
+    title: 'test-title',
+    owner: 'test-owner',
+    category: 'test-type',
+    city: 'test-city',
+    bedrooms: 3
+  });
```

Comment:
Since we can't guarantee that nothing mutates that object we should create a new one for each test in a beforeEach block